cli/providers/aws/translate.go

```diff
@@ -301,7 +301,7 @@ func loadELB(t *gocf.Template, s *service.Service, e *environment.Environment) e
 		crossZone = true
 	}
 	elb := &gocf.ElasticLoadBalancingLoadBalancer{
-		LoadBalancerName: gocf.String(s.GetFullName(e, "-")),
+		LoadBalancerName: gocf.String(s.GetFullName(e, "-")[:32]),
```

Comment:
Lets truncate some chars in the service name and some others in the environment name to avoid collisions.

Reply:
I have a separate PR to remove the ELB name, I think we should just go with that one instead (giving an ELB a name prevents it from being replaced during an upgrade)

Reply:
Right, I approved the other one, closing this...